ingest/sqs/service/writelistener/concentrated_write_listener_test.go

```diff
@@ -0,0 +1,123 @@
+package writelistener_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v24/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v24/ingest/sqs/service"
+	"github.com/osmosis-labs/osmosis/v24/ingest/sqs/service/writelistener"
+	"github.com/osmosis-labs/osmosis/v24/x/concentrated-liquidity/model"
+	concentratedtypes "github.com/osmosis-labs/osmosis/v24/x/concentrated-liquidity/types"
+)
+
+type WriteListenerTestSuite struct {
+	apptesting.ConcentratedKeeperTestHelper
+}
+
+var (
+	someValue = []byte("someValue")
+)
+
+func TestWriteListenerTestSuite(t *testing.T) {
+	suite.Run(t, new(WriteListenerTestSuite))
+}
+
+// Tests that the concentrated write listener correctly tracks pool and tick updates
+// It ignores error cases as they are simple and unlikely giving app wiring.
+func (s *WriteListenerTestSuite) TestWriteListener_Concentrated() {
+
+	// Set up chain state once per test
+	s.Setup()
+
+	concentratedPool := s.PrepareConcentratedPool()
+	concentratedPoolModel, ok := concentratedPool.(*model.Pool)
+	s.Require().True(ok)
+
+	concentratedPoolModelBz, err := concentratedPoolModel.Marshal()
+	s.Require().NoError(err)
+
+	testCases := []struct {
+		name string
+
+		key      []byte
+		value    []byte
+		isDelete bool
+
+		expectedPoolUpdate     bool
+		expectedPoolTickUpdate bool
+	}{
+		{
+			name: "random concentrated write, no-op",
```

Comment:
nit: In these tests by "random" I think its better said "concentrated write unrelated to pool state"

Reply:
changed per suggestion for all pools